internal/pay/data.go

```diff
@@ -68,13 +68,13 @@ type RefundSummary struct {
 }
 
 type SettlementSummary struct {
-	CaptureSubmitTime string `json:"capture_submit_time"`
-	CapturedDate      string `json:"captured_date"`
-	SettledDate       string `json:"settled_date"`
+	CaptureSubmitTime date.Date `json:"capture_submit_time"`
+	CapturedDate      date.Date `json:"captured_date"`
+	SettledDate       date.Date `json:"settled_date"`
 }
 
 type GetPaymentResponse struct {
-	CreatedDate time.Time `json:"created_date"`
+	CreatedDate date.Date `json:"created_date"`
```

Comment:
![image](https://github.com/user-attachments/assets/c9a3cd2a-226f-4acf-b3a8-f63296a9d059)

and this is `time.Time` too